src/Components/Card.js

```diff
@@ -36,7 +37,7 @@ const Card = ({
         <CardLi>
           <img src={image} alt="Profile picture" />
           <h2>{name}</h2>
-          { bookmarking ? (<Bookmark className={isActive ? "active" : null} onClick={() => {setActive(!isActive); handleBookmarking(id)}}></Bookmark>) : ""}
+          { bookmarking ? (<Bookmark className={isbookmarked ? "activebookmark" : ""} onClick={() => {handleBookmarking(id)}}></Bookmark>) : ""}
```

Comment:
It would be slightly cleaner if the handleBookmarking function here would not consume an id as a parameter. The components using the Card component already know which id should be used inside this function.

Reply:
Hi Thomas, I don't know how to do this. If I just remove the parameter, it doesn't work anymore and I don't understand how it should even work without this information....

Reply:
In the FavoritesPage component for example you provide the prop like this:
`handleBookmarking={handleBookmarking}`
This part would have to be changed so the provided function doesn't need an argument anymore

Reply:
That is how I provide the prop in the Details Page as well. However when I call the function itself from the card, I pass the ID. From my point of view this is needed. I will move on for now but if you think that I still didn't get you right maybe we can discuss it another time?